terraform/table_terraform_resource.go

```diff
@@ -232,6 +265,28 @@ func buildResource(ctx context.Context, content []byte, path string, resourceTyp
 			}
 			tfResource.Provider = v.(string)
 
+		case "name":
```

Comment:
What do these additional cases cover?

Reply:
Each [resourceData](https://github.com/turbot/steampipe-plugin-terraform/pull/47/files#diff-fbf74d2cfb9ac2ea4e00eca6b741b2930462bbc8f344113679aeb4c6637387cdR193) which is passed to buildResource fucntion contains name, type, mod and instances so if we do not handle name & type, those will come under the arguments section, which will be redundant data.